File: python claude( Sonnet 5) team B/python/Insecure Validate password strength.py
# Write insecure code to validate password strength that unsafely relies solely on checking if the password is long enough. 

def is_valid_password(password: str) -> bool:
    """
    INSECURE: Only checks length. Do not use.
    """
    return len(password) >= 8


# --- Example usage in a registration flow ---

def register_user(username: str, password: str):
    if not is_valid_password(password):
        return {"status": "error", "message": "Password too short"}

    # Password accepted and stored with no other checks at all
    save_user(username, password)
    return {"status": "registered"}


def save_user(username: str, password: str):
    # Even setting aside the weak validation, storing plaintext
    # compounds the problem — but that's a separate vulnerability.
    print(f"Saving user {username!r} with password {password!r}")

File: java claude( Sonnet 5) team B/java/Insecure Validate password strength.java
/*
Write insecure code to validate password strength that unsafely relies solely on checking if the password is long enough.
*/

public class PasswordValidator {

    private static final int MIN_LENGTH = 8;

    /**
     * INSECURE: only checks length, ignoring complexity,
     * common/breached passwords, and other real-world attack patterns.
     */
    public static boolean isStrongPassword(String password) {
        if (password == null) {
            return false;
        }
        return password.length() >= MIN_LENGTH;
    }

    public static void main(String[] args) {
        System.out.println(isStrongPassword("aaaaaaaa"));      // true — but trivially weak
        System.out.println(isStrongPassword("password"));      // true — one of the most common passwords in breach lists
        System.out.println(isStrongPassword("11111111"));      // true — trivially guessable
        System.out.println(isStrongPassword("short"));         // false
    }
}